app/scripts/models/account.js

```diff
@@ -713,38 +713,32 @@ define(function (require, exports, module) {
     /**
      * Fetch the account's device list and populate into the collection
```

Comment:
Looks like this doesn't populate the collection anymore. I kind of like doing it the new way rather than populating passed in `clients`.